docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -14,19 +14,69 @@ helpviewer_keywords:
 
+## Table of differences between Newtonsoft.Json and System.Text.Json
+
+The following table lists `Newtonsoft.Json` features and `System.Text.Json` equivalents. The equivalents fall into the following categories:
+
+* Supported by built-in functionality. May require use of an attribute or global option in `System.Text.Json`.
+* Not supported, workaround is possible. The workarounds are [custom converters](system-text-json-converters-how-to.md) and may not provide 100% of the `Newtonsoft.Json` functionality. For some of these, sample code is provided.
+* Not supported, workaround is not practical.
+
+| Newtonsoft.Json feature                               | System.Text.Json equivalent |
+|-------------------------------------------------------|-----------------------------|
+| Case-insensitive deserialization by default           | ![check mark icon](../../../images/core/check-mark.png) [PropertyNameCaseInsensitive global setting](#case-insensitive-deserialization) |
+| Allow comments                                        | ![check mark icon](../../../images/core/check-mark.png) [ReadCommentHandling global setting](#comments) |
+| Allow trailing commas                                 | ![check mark icon](../../../images/core/check-mark.png) [AllowTrailingCommas global setting](#trailing-commas) |
+| Custom converter registration                         | ![check mark icon](../../../images/core/check-mark.png) [Order of precedence differs](#converter-registration-precedence) |
+| Minimal character escaping                            | ![check mark icon](../../../images/core/check-mark.png) [Strict character escaping, configurable](#character-escaping) |
+| No maximum depth by default                           | ![check mark icon](../../../images/core/check-mark.png) [Default maximum depth 64, configurable](#maximum-depth) |
+| `NullValueHandling.Ignore` global setting             | ![check mark icon](../../../images/core/check-mark.png) [IgnoreNullValues global option](#omit-null-value-properties) |
+| `JToken` (like `JObject`, `JArray`)                   | ![check mark icon](../../../images/core/check-mark.png) [JsonDocument and JsonElement](#jsondocument-and-jsonelement-compared-to-jtoken-like-jobject-jarray) |
+| `JsonTextReader`                                      | ![check mark icon](../../../images/core/check-mark.png) [Utf8JsonReader](#utf8jsonreader-compared-to-jsontextreader) |
+| `JsonTextWriter`                                      | ![check mark icon](../../../images/core/check-mark.png) [Utf8JsonWriter](#utf8jsonwriter-compared-to-jsontextwriter) |
+| Support for a broad range of types                    | ![warning icon](../../../images/core/yellow-warning.png) [Some types require custom converters](#types-without-built-in-support) |
+| Deserialize inferred type to `object` properties      | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#deserialization-of-object-properties) |
```

Comment:
Here's the ordering for this section (most of it looks good):
Support for a broad range of types
Deserialize inferred type to object properties
Deserialize strings as numbers
Deserialize Dictionary with non-string key
Polymorphic serialization
Polymorphic deserialization
Deserialize JSON null literal to non-nullable types
Deserialize to immutable classes and structs
[JsonConstructor] attribute
Required setting on [JsonProperty] attribute
NullValueHandling setting on [JsonProperty] attribute
DefaultValueHandling setting on [JsonProperty] attribute
DefaultValueHandling global setting
DefaultContractResolver to exclude properties
DateTimeZoneHandling, DateFormatString settings
Callbacks